### DISCUSSION TITLE
N64 hud in 3ds is small

### DISCUSSION THREAD

**Discussion by ghost** [leoetlino/project-restoration#224] [Q&A] [answered]
I put the files to have the custom hud in my 3ds and it works but it is very small, how can it be fixed?
![16549182614496107992219952100924](https://user-images.githubusercontent.com/84218863/173170890-e432d1ec-5f61-44ab-8652-e9b4a1dbae6c.jpg)

**Comment by leoetlino** [comment, marked as answer]
The HD HUD is only intended for use on HD screens. The 3DS' screen is way too low-res for the mod to work well.

While it technically runs on 3DS consoles just fine, it is not officially supported on anything but Citra.

**Reply by ghost** [reply]
ok but can't it be made bigger to use it in 3ds?

**Reply by leoetlino** [reply]
Technically it could -- but it won't look good. There simply aren't enough pixels to render widgets at a smaller size (compared to the original sizes on the bottom screen). Plus it would significantly clutter the top screen.

**Reply by ghost** [reply]
ok but I don't care if it looks good all I want is to know how to change the size
